lineman/app/components/start_menu/start_menu.coffee

```diff
@@ -0,0 +1,12 @@
+angular.module('loomioApp').directive 'startMenu', ->
+  scope: {}
+  restrict: 'E'
+  templateUrl: 'generated/components/start_menu/start_menu.html'
+  replace: true
+  controller: ($scope, $modal) ->
+
+    $scope.$on 'modalOpened', ->
+      $scope.startMenuOpen = false
```

Comment:
We want to know when the modal opens, so we can close the menu.

It strikes me there may be other cases where globally notifying the app that a modal is opening might be useful.